src/test/java/com/github/timm/cucumber/options/RuntimeOptionsTest.java

```diff
@@ -68,12 +64,6 @@ public void assigns_strict_short() {
     }
 
     @Test
-    public void default_strict() {
```

Comment:
No longer needed. .strict is enabled by default in the GenerateRunnersMojo. Because strict is now passed as an object rather then a primitive this won't override the strict setting from GenerateRunnersMojo unless it has been explicitly provided on the commandline.